azkaban-common/src/main/java/azkaban/executor/ConnectorParams.java

```diff
@@ -80,12 +79,33 @@
   public static final String JMX_GET_MBEANS = "getMBeans";
   public static final String JMX_GET_MBEAN_INFO = "getMBeanInfo";
   public static final String JMX_GET_MBEAN_ATTRIBUTE = "getAttribute";
-  public static final String JMX_GET_ALL_MBEAN_ATTRIBUTES =
-      "getAllMBeanAttributes";
+  public static final String JMX_GET_ALL_MBEAN_ATTRIBUTES = "getAllMBeanAttributes";
   public static final String JMX_ATTRIBUTE = "attribute";
   public static final String JMX_MBEAN = "mBean";
 
-  public static final String JMX_GET_ALL_EXECUTOR_ATTRIBUTES =
-      "getAllExecutorAttributes";
+  public static final String JMX_GET_ALL_EXECUTOR_ATTRIBUTES = "getAllExecutorAttributes";
   public static final String JMX_HOSTPORT = "hostPort";
+
+  public static final String STATS_GET_ALLMETRICSNAME = "getAllMetricNames";
+  public static final String STATS_GET_METRICHISTORY = "metricHistory";
```

Comment:
does it make sense to be consistent in the naming? metricHistory -> getMetricHistory